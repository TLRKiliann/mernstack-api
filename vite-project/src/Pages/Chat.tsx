import { Component } from 'react'
import '../stylePages/Chat.scss'

export default class Chat extends Component {
  render() {
    return(
      <div>
        Chat
      </div>
    )
  }
}