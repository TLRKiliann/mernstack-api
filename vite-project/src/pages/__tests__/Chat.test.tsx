import React from "react";
import {assert, assertType, expectTypeOf, beforeEach, afterEach, describe, expect, test, it, vi} from 'vitest';
import {fireEvent, render, screen} from '@testing-library/react';
//import "@testing-library/jest-dom/extend-expect"
//import "@testing-library/jest-dom";
//import { act } from 'react-dom/test-utils';
import {create} from 'react-test-renderer';
import Chat from "../Chat.tsx";

test('MatchSnapShot test Chat', () => {
	const tree = create(<Chat />)
	expect(tree.toJSON()).toMatchSnapshot();
})