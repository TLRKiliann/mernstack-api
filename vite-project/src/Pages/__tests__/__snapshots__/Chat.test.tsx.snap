// Vitest Snapshot v1

exports[`MatchSnapShot test Chat 1`] = `
<div>
  <div
    className="title--chat"
  >
    <h1>
      Chat
    </h1>
  </div>
</div>
`;
